packages/collection-model/lib/model.js

```diff
@@ -269,18 +269,21 @@ const CollectionModel = AmpersandModel.extend(debounceActions(['fetch']), {
       // We don't care if it fails to get stats from server for any reason
     }
 
+    const instance = getParentByType(this, 'Instance');
+
     /**
      * The support for search indexes is a feature of a server not a collection.
      * As this check can only be performed currently by running $listSearchIndexes
      * aggregation stage against a collection, so we run it from the collection model.
      * With this setup, when a user opens the first collection, we set this property
      * on the instance model and then from there its value is read avoiding call to server.
      */
-    const isSearchIndexesSupported = await getParentByType(this, 'Instance')
-      .getIsSearchSupported({
+    const isSearchIndexesSupported =
+      !this.isView &&
```

Comment:
Was it previously a bug showing them on view or you just moved this check from somewhere else?

Reply:
This would always fail for views because you can't have indexes for views, this wasn't causing anything in the UI, but it's just unneccessary to fetch this information (it's a sequential request, so we can safely skip it and make this fetch faster for views)